Firebase/Auth/Source/Public/FIROAuthProvider.h

```diff
@@ -23,17 +23,17 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-
 /**
-    @brief A string constant identifying the Microsoft identity provider.
+    @brief Please use NSString @"microsoft.com" instead.
  */
-extern NSString *const FIRMicrosoftAuthProviderID NS_SWIFT_NAME(MicrosoftAuthProviderID);
+extern NSString *const FIRMicrosoftAuthProviderID
+    NS_SWIFT_NAME(MicrosoftAuthProviderID) __attribute__((deprecated));
```

Comment:
Instead of `__attribute__((deprecated))` can you please use `DEPRECATED_MSG_ATTRIBUTE("Please use \"microsoft.com\" instead.")`

and the same below but for Yahoo? Then leave the `brief` since it's still useful. This will give users a warning if they have it, with instructions in the warning.

Reply:
Done.